docs(player-support): add Yandex Station player provider page (#625)
music-assistant/music-assistant.io: src/content/docs/player-support/yandex-station.md

- **QR code** — Scan the QR shown during setup with the Yandex app (or phone camera) and confirm the login. No refresh token: the session must be renewed manually when it expires.
- **Cookies (advanced fallback)** — Paste Yandex session cookies as a JSON array exported from browser dev tools. Use when the other two methods are blocked by your network or account.

After login, stations on the same network are auto-discovered via mDNS and registered as players.

### Settings

- **Remember session** — store the refresh token / tokens so the provider survives MA restarts without re-authentication. On by default.
- **Voice control integration** *(per-player, advanced)* — off by default. When enabled, the provider detects when you talk to Alice during MA playback and reacts accordingly (see below).

### Voice control integration (experimental)

When enabled in **Settings → Players → \<Your Station\> → Show advanced → Voice control integration**, the provider watches the Glagol state stream and reacts to Alice activity:

| Voice command | Behaviour |
|---|---|

> | «Алиса, стоп» | Pauses the Music Assistant queue (resume via the UI). |

Reviewer: Does this provider only work for Russian speaking users?

Author: No,  in english too. Fixed

| «Алиса, какая погода?» *(or any query)* | Pauses MA while Alice speaks, then auto-resumes playback. |
| «Алиса, громче / тише» | Adjusts volume during playback and auto-resumes the queue. |

## Known Issues / Notes

- Stations must reach the Music Assistant stream server over plain **HTTP** — HTTPS URLs for local addresses are rejected by the station. Make sure the [Stream server base URL](/settings/core/#streams) is an `http://` URL on the LAN.

> - The MA stream server must serve a valid `Content-Length` header. Chunked transfer encoding is not accepted by the station and results in silence.

Reviewer: What can the user do with this information? Same comment for the next line

Author: fixed

- Stream URLs must carry a file extension the station recognises (`.flac`, `.mp3`). MA's `resolve_stream_url()` already provides this.
- Voice commands «дальше» / «назад» cannot advance the MA queue — Alice does not see the external stream bypass Music Assistant uses, so the station's own "next / previous" targets Alice's content, not the MA queue. Use the MA UI or other voice assistants for queue navigation.
- Seek via voice is not supported for the same reason.
- The Glagol "stop" command behaves as "pause" on the station (the station keeps the current track loaded). The provider treats both MA `pause()` and `stop()` as a Glagol pause.

> - Power on/off relies on Yandex scenarios created in the Yandex app; if scenarios are unavailable for a given station model, the power button in MA will be a no-op.

Reviewer: ```suggestion
- Power on/off relies on Yandex scenarios created in the Yandex app; if scenarios are unavailable for a given station model, the power button in MA will not work .
```
